lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu 
+    
+    MENU_ITEMS = 
+        { 
```

Comment:
Style/IndentAssignment: Indent the first line of the right-hand-side of a multi-line assignment.<br>Style/MutableConstant: Freeze mutable objects assigned to constants.